tests/ci/ci_config.py

```diff
@@ -637,16 +637,8 @@ def validate(self) -> None:
         Labels.CI_SET_INTEGRATION: LabelConfig(
             run_jobs=[
                 JobNames.STYLE_CHECK,
-                Build.PACKAGE_ASAN,
```

Comment:
Unrelated changes?

Reply:
just a bit related. i use #ci_set_imtegration tag to test my changes in ci. i made this tag to run only "integration tests (release)"